docs/tutorials/stateful-application/cassandra.md

```diff
@@ -39,343 +39,147 @@ import it to the website.
-  - [Step 2: Use a StatefulSet to create Cassandra Ring](#step-2-use-a-statefulset-to-create-cassandra-ring)
-  - [Step 3: Validate and Modify The Cassandra StatefulSet](#step-3-validate-and-modify-the-cassandra-statefulset)
-  - [Step 4: Delete Cassandra StatefulSet](#step-4-delete-cassandra-statefulset)
-  - [Step 5: Use a Replication Controller to create Cassandra node pods](#step-5-use-a-replication-controller-to-create-cassandra-node-pods)
-  - [Step 6: Scale up the Cassandra cluster](#step-6-scale-up-the-cassandra-cluster)
-  - [Step 7: Delete the Replication Controller](#step-7-delete-the-replication-controller)
-  - [Step 8: Use a DaemonSet instead of a Replication Controller](#step-8-use-a-daemonset-instead-of-a-replication-controller)
-  - [Step 9: Resource Cleanup](#step-9-resource-cleanup)
-  - [Seed Provider Source](#seed-provider-source)
-
-The following document describes the development of a _cloud native_
-[Cassandra](http://cassandra.apache.org/) deployment on Kubernetes.  When we say
-_cloud native_, we mean an application which understands that it is running
-within a cluster manager, and uses this cluster management infrastructure to
-help implement the application.  In particular, in this instance, a custom
-Cassandra `SeedProvider` is used to enable Cassandra to dynamically discover
-new Cassandra nodes as they join the cluster.
-
-This example also uses some of the core components of Kubernetes:
-
-- [_Pods_](/docs/user-guide/pods)
-- [ _Services_](/docs/user-guide/services)
-- [_Replication Controllers_](/docs/user-guide/replication-controller)
-- [_Stateful Sets_](/docs/concepts/workloads/controllers/statefulset/)
-- [_Daemon Sets_](/docs/admin/daemons)
-
-## Prerequisites
-
-This example assumes that you have a Kubernetes version >=1.2 cluster installed and running,
-and that you have installed the [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-command line tool somewhere in your path.  Please see the
-[getting started guides](https://kubernetes.io/docs/getting-started-guides/)
-for installation instructions for your platform.
-
-This example also has a few code and configuration files needed.  To avoid
-typing these out, you can `git clone` the Kubernetes repository to your local
-computer.
-
-## Cassandra Docker
+{% capture overview %}
+This tutorial shows you how to develop a native cloud [Cassandra](http://cassandra.apache.org/) deployment on Kubernetes. In this instance, a custom Cassandra `SeedProvider` enables Cassandra to discover new Cassandra nodes as they join the cluster.
 
+Deploying stateful distributed applications, like Cassandra, within a clustered environment can be challenging. StatefulSets greatly simplify this process. Please read about [StatefulSets](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/)  for more information about the features used in this tutorial.
+
+**Cassandra Docker**
```

Comment:
if you need it to stay on its own line, you need to add `<br/>` explicitly at the end of the line, however I would recommend using a `####` (h4) heading.